includes/wikia/models/NavigationModel.class.php

```diff
@@ -453,62 +420,62 @@ public function parseLines($lines, $maxChildrenAtLevel = array()) {
 					if ( $depth == $lastDepth + 1 ) {
 						$parentIndex = $i;
 					} else if ( $depth == $lastDepth ) {
-						$parentIndex = $nodes[$i][ self::PARENT_INDEX ];
+						$parentIndex = $nodes[$i][self::PARENT_INDEX];
 					} else {
-						for( $x = $i; $x >= 0; $x-- ) {
+						for ( $x = $i; $x >= 0; $x-- ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS134)